datasets/cellpainting-gallery.yaml

```diff
@@ -38,36 +38,36 @@ DataAtWork:
   Tutorials:
     - Title: Cell Painting wiki
       URL: https://github.com/carpenterlab/2016_bray_natprot/wiki
```

Comment:
We've decided in other places to use broad.io/cellpaintingwiki so that we can update to the new paper's repo freely

Reply:
Nice, please do 

Also, please close this when you do https://github.com/broadinstitute/cellpainting-gallery/issues/8